controllers/topics.controller.js

```diff
@@ -0,0 +1,11 @@
+const fetchTopics = require("../models/topics.model")
+
+function getTopics(req, res, next) {
+    fetchTopics()
+    .then((topics) => {
+        res.status(200).send(topics.rows)
```

Comment:
We should send back data to the client on a meaningful key e.g, they should be able to access it on body.topics :)